.gitattributes

```diff
@@ -4,6 +4,7 @@ tools/crunch-worker.js -diff
 third_party/lzma.js/lzma-decoder.js -diff
 third_party/lzma.js/lzma-full.js -diff
 tests/other/test_emsize.js -diff
+src/emscripten-source-map.min.js -diff
```

Comment:
We add minified files to this list of prevent them showing up in `git grep` etc.   Without this a `git grep source-map` will print the single line of this minified file which is thousands of chars long.